README.md

```diff
@@ -1,2 +1,22 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a)go through two lists then see if the list are different sizes but have the same final number
+
+(b)N has to go through two loops which causes it to have an O(n^2)
+
+(c)changing line 4 will always make the first condition true and dosn't make the second condition false
+
+(d)N only has to go through one loop
+
+(e) no as i said above N only has to go through one loop
+
+(f) Worst case performance  O(nlogn) Best case performance   O(n) 
+Average case performance    O(nlogn) Worst case space complexity O(n) 
+https://stackoverflow.com/questions/25813774/complexity-of-python-sort-method
+
+(g) O(n) as it has a sort function
```

Comment:
But O(n log n) dominates O(n), therefore the time complexity is O(n log n). Remember that time complexity considers the worst case, not the best case